src/Pages/Detail/Components/ImgContainer.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const ImgContainer = ({ imgSrc }) => {
+  return (
+    <StyledImgContainer>
+      <BigImg>
+        <img className="firstImg" alt="thumbnail" src={imgSrc} />
+      </BigImg>
+      <SmallImgContainer>
+        <SmallImg>
+          <img
+            className="firstImg"
+            alt="thumbnail"
+            src="images/marcia-bartho-Uasj0wDn6AQ-unsplash.jpg"
+          />
+        </SmallImg>
+        <SmallImg>
+          <img
+            className="secondImg"
+            alt="thumbnail"
+            src="images/spacejoy-LhOJl195TWc-unsplash.jpg"
+          />
+        </SmallImg>
+        <SmallImg>
+          <img
+            className="thirdImg"
+            alt="thumbnail"
+            src="images/florian-douillet-9Nig3VNWP-Q-unsplash.jpg"
+          />
+        </SmallImg>
+        <SmallImg>
+          <img
+            className="fourthImg"
+            alt="thumbnail"
+            src="images/cater-yang-1a_u4n02YNo-unsplash.jpg"
+          />
+        </SmallImg>
+      </SmallImgContainer>
+    </StyledImgContainer>
+  );
+};
+
+const StyledImgContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 50px;
+`;
+
+const BigImg = styled.div`
+  width: 49.5%;
+
+  img {
+    width: 100%;
+    height: 557px;
+    object-fit: cover;
+    border-top-left-radius: 15px;
+    border-bottom-left-radius: 15px;
+  }
+`;
+
+const SmallImgContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 49.5%;
+`;
+
+const SmallImg = styled.div`
+  width: 49%;
+
+  img {
+    width: 100%;
+  }
+
+  .secondImg {
+    border-top-right-radius: 15px;
+  }
+
+  .fourthImg {
+    border-bottom-right-radius: 15px;
+  }
```

Comment:
그리고 이 케이스에는 SmallImg 자체에 가상선택자 사용해서 스타일링 하는게 더 목적에 맞아보입니다!
```suggestion
  $:nth-child(2) {
    img{
      border-top-right-radius: 15px;
    }
  }